rdkit/ML/ScreenComposite.py

```diff
@@ -706,25 +698,23 @@ def ScreenFromDetails(models, details, callback=None, setup=None, appendExamples
                                 pickleClass=DataStructs.ExplicitBitVect)
     else:
       data = details.GetDataSet()
-  if details.threshold > 0.0:
-    details.partialVote = 1
-  else:
-    details.partialVote = 0
+  
+  details.partialVote = int(bool(details.threshold > 0.0))
```

Comment:
I'm not sure I like this reformatting.

The "pythonic" way is
```
details.partialVote = 1 if details.threshold > 0.0 else 0
```

Reply:
I got your idea! I would fix it in the next patch. Thank you for your review.